### DISCUSSION TITLE
Error when trying to use StringUtils

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5410] [Q&A]
I have a code template that uses 

```
var StringUtils = org.apache.commons.lang3.StringUtils;
```
It is working fine in existing channels.  I just tried cloning one of the channels and now get this error for the same code:

 TypeError: Cannot read property "commons" from undefined

Any ideas why the cloned channel isn't working correctly?

**Comment by jonbartels** [comment]
What MC versions are in use?

Show your channel dependencies, are you depending on the "default resource"?

Does `Packages.org.apache.commons.lang3.StringUtils` change the behavior?
